src/Symfony/Component/HttpKernel/Exception/HttpException.php

```diff
@@ -21,6 +21,15 @@ class HttpException extends \RuntimeException implements HttpExceptionInterface
     private $statusCode;
     private $headers;
 
+    /**
+     * HttpException constructor.
```

Comment:
We can remove this line --> `HttpException constructor.` because it's not useful for constructors.